### ISSUE TITLE
[BALANCE] Mystic Arms and Bows

### ISSUE THREAD

**Issue by cambragol** [cambragol/advent-of-the-trinity#251]
Should they be the only thing that works in the Abyss?

I remember feeling a bit cheesed when I discovered that I didn't need the mystic arms, and was much better served in the abyss with magic axes and wands.

The wording of things seemed to imply that the mystic arms were required, when actually, they were not.

I am thinking of forcing this. Having the Mystic Bows would also prevent any whining about ranged combat being lost because of the restriction of Magic weapons in the Abyss

**Comment by cambragol** [comment]
Regarding the Mystic Arms:

"Then seek out
the smith named
Zircon in Minoc
for he made the
mystic arms,
**only they** will
save thee in the
abyss!"

**Comment by MrKnitster** [comment]
Agreed. The game definitely feels like it's pushing you in that direction. I think they should be the only effective weapons. Maybe even a HELP comment from Lord British when he tells you you're ready to enter. He could say:

"Thou dost now seem ready to make the final journey into the dark Abyss! Go only with a party of eight! **You must have the mystic arms!** Good Luck, and may the powers of good watch over thee on this thy most perilous endeavor! The hearts and souls of all Britannia go with thee now. Take care, my friend..."

**Comment by cambragol** [comment]
I am definitely leaning this way as well. I originally created the Mystic Bows, to make 'mystic weapons' that people would want to use in the Abyss. However, my effort is wasted, somewhat, if everyone still prefers the magic wand, axe or bow.

Maybe in original playtesting they decided that forcing the players to suddenly resort to melee only, after playing a mixed melee/range game up until that point, was unfair.

My Mystic Bows fixes this, so perhaps the 'original' limitation, can be restored.

The hint from Lord British, or the underscore, is a good idea too.

**Comment by cambragol** [comment]
The 'too close' limitation for ranged weapons also gives incentive to get not only the bows, but also the arms. So both quests are useful/critical again. Mystic Bow could still be skipped, as technically you could do the Abyss with mystic arms only.

**Comment by cambragol** [comment]
I think the main argument against this would be that you can sell both the Mystic Arms and Bows, and then will never be able to recover them again, and your game is over.

But...wouldn't that just serve you right? You were told it was the only weapon that would work in the Abyss. You go on a sacred mission to find them, then callously sell them for cash?!?

Not much sympathy from me.

**Comment by Fenyx4** [comment]
I like the idea of making only Mystics work. (Although I will say that I saw someone quit the game entirely after going into the Abyss and not having their non-magical weapons work. But that was a weird case.)

Could someone also lose the Mystics from not having them equipped and dying? I know in my playthrough I lost some weapons (not the Mystics though) because I was swapping between ranged and melee on characters and then died.

**Comment by cambragol** [comment]
Hmm...that could happen.  Especially with the need to switch between ranged and melee weapons. It is very possible the Mystics, either type, could be lost on death. No easy fix for that. Which is probably why magic weapons were allowed to work in the Abyss in the first place.

A fix for this would be to have LB recover mystic weapons on death.

**Comment by cambragol** [comment]
It is very simple to have the mystic weapons and armor kept on death, which would make sense if they are mission critical, like the candle, book, key of 3 parts etc., which are all kept on dying.

But what of reagents and mixtures...?

**Comment by Fenyx4** [comment]
That would be resetting us back to square zero. 😭

Although it would make non mages more valuable...

But overall I think it would make dying too harsh a punishment. I am guessing people would just revert to their last save to avoid it.

**Comment by cambragol** [comment]
Yeah, a little harsh. Compared to default. But not really harsh. You did die after all. I've always thought the punishments for dying in Ultima IV were quite gentle. Ultima V is certainly worse. Heck, you get brought back to life after every death!

I don't think I want or need to fiddle with reagents and mixtures though. Just preventing the Mystics from being lost enough. Paired with mystics only in the Abyss, as per the game's lore.

**Comment by cambragol** [comment]
Just tried randomizing the reagents and mixtures. Lord British brings back *some* of your belongings from the void. You get back a random quantity, the max of which is your current total just at death. This seems like a nice tweak.

**Comment by cambragol** [comment]
This is fixed thusly in 13.9

Mystic Bows, Weapons and Armor can not be lost on death. They are brought back from the void by LB.
Reagents

Other items are unaffected and as per default.

**Comment by cambragol** [comment]
Tested. Only Mystic Bows and Mystic Arms working in Abyss. Magic weapons always miss, as do normal weapons.

After death, unequipped mystic weapons were restored, unequipped magic weapons and normal were lost.
